Source/Atomic/Atomic2D/ParticleEmitter2D.h

```diff
@@ -88,6 +88,8 @@ class ATOMIC_API ParticleEmitter2D : public Drawable2D
     /// Handle enabled/disabled state change.
     virtual void OnSetEnabled();
 
+	/// Set emission enabled.
+	void SetEmissionEnabled(bool enable);
     /// Set particle effect.

```

Comment:
Look great, and thanks for adding /// comments, these will automatically show up in C# and TS.